lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -621,7 +621,9 @@ static void topK(Tensor &outW, Tensor &indW, Tensor &inW, size_t k) {
   }
 }
 
-void OpenCLFunction::execute() {
+void OpenCLFunction::execute(Context *ctx) {
```

Comment:
you can just do: execute (Context *) and remove (void)ctx.

Reply:
heh, @opti-mix has a comment asking for the reverse above. Personally, not worried either way.

Reply:
ok, does not matter indeed.